src/diffusers/pipelines/pixart_alpha/pipeline_pixart_alpha.py

```diff
@@ -228,6 +264,7 @@ def __init__(
         vae: AutoencoderKL,
         transformer: Transformer2DModel,
         scheduler: DPMSolverMultistepScheduler,
+        model_token_max_length: int = 120,
```

Comment:
@yiyixuxu why did we decide to make this as a config variable instead of a pipeline call arg?

Reply:
@sayakpaul @lawrence-cj 
oh I'm not sure 
Is this something we need to change each generation? e.g., if we need to adjust this value based on the prompt, I think it makes sense to add it to pipeline call arg; Otherwise, we can add it as a config, no?

Reply:
The maximum sequence length bit should be something a user wants to experiment with. I don't think it needs to be a configuration variable.

Reply:
ok!